using_images/db_images/mariadb.adoc

```diff
@@ -88,7 +88,7 @@ xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with
 MariaDB running in a container:
 
 ----
-$ oc new-app \
+$ docker run -d --name mariadb_database \
     -e MYSQL_USER=<username> \
     -e MYSQL_PASSWORD=<password> \
     -e MYSQL_DATABASE=<database_name> \
```

Comment:
I think it is no need to modify command, just like below is ok:
`$ oc new-app \
    -e MYSQL_USER=<username> \
    -e MYSQL_PASSWORD=<password> \
    -e MYSQL_DATABASE=<database_name> \
    centos/mariadb-101-centos7`